common/deployment/test_scenarios.sh

```diff
@@ -35,107 +35,53 @@ declare -A test_scenario1=(
     [skip]=false
     [modes]="FULL QUICK PUBLISH"
 )
-declare -A test_scenario2=(
-    [name]="02-oidc_password_grant"
-    [display_name]="OIDC Password Grant Type"
-    [description]="Obtain an access token and an id token using the OAuth 2.0 password grant type."
-    [jmx]="oidc/OIDC_Password_Grant.jmx"
-    [tenantMode]=false
-    [skip]=false
-    [modes]="FULL QUICK PUBLISH"
-)
-declare -A test_scenario3=(
-    [name]="03-saml2_sso_redirect_binding"
-    [display_name]="SAML2 SSO Redirect Binding"
-    [description]="Obtain a SAML 2 assertion response using redirect binding."
-    [jmx]="saml/SAML2_SSO_Redirect_Binding.jmx"
+declare -A test_scenario1=(
```

Comment:
Update the correct number order

Reply:
Fixed with 85e1f400c043d2e50b0c91c41e0419b5696715d8

Reply:
Fixed with c4307608819dea0e6575e951f61f3efeeea61e17